### ISSUE TITLE
Article details: Author not visible with large text

### ISSUE THREAD

**Issue by antonis** [wordpress-mobile/WordPress-iOS#21834]
### Expected behavior
On small screen or with large text/display configuration the author of the post is visible in the post details screen

### Actual behavior
On small screen or with large text/display configuration the author of the post is hidden bellow the date/time in the article details screen

### Steps to reproduce the behavior
1. Set a large font size or the large text display zoom
2. Browse at an article detail
3. Notice that the author is not visible

|Article 1|Article 2|
|---|---|
|![IMG_0209](https://github.com/wordpress-mobile/WordPress-iOS/assets/304044/e8ee9e8a-ea85-4121-856a-0a4c7def83d5)|![IMG_0214](https://github.com/wordpress-mobile/WordPress-iOS/assets/304044/f872347e-416f-4a9b-bc88-578aa5b93406)|

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by kean** [comment]
It's partially fixed in https://github.com/wordpress-mobile/WordPress-iOS/pull/23909, but this page still has multiple outstanding dynamic type support issues.
